java/src/com/ibm/streamsx/topology/Topology.java

```diff
@@ -698,4 +699,68 @@ public GraphBuilder builder() {
 
         return names;
     }
+
+    /**
+     * Create a submission parameter without a default value.
+     * <p>
+     * A submission parameter is a handle for a {@code T} whose actual value
+     * is not defined until topology execution time.
+     * <p>
+     * A submission parameter has a {@code name}.  The name must be unique
+     * within the topology.
+     * <p>
+     * The returned Supplier's {@code get()} returns null.
+     * <p>
+     * Submission parameters may be used for values in various
+     * cases such as {@code TStream.parallel()} width specifications.
+     * e.g.,
+     * <pre>{@code
+     * TStream<Foo> s = ...
+     * s.parallel(top.createSubmissionParameter(..., 5) ... // default width of 5
+     * s.parallel(top.createSubmissionParameter(..., Integer.class) ...
```

Comment:
Somewhat confusing having two parallel calls against the same stream, and maybe too many '...', leaving too much guesswork for the reader.

Why not just have a complete example?